src/jobs/scratch-deploy-and-test.yml

```diff
@@ -1,6 +1,6 @@
 description: >
```

Comment:
This job must support all of the parameters for each command within, including the two new wait time parameters.